en/console-commands/commands.rst

```diff
@@ -255,6 +255,7 @@ You may want to set a command description via::
 
 This will show your description in the Cake CLI::
 
+... code-block:: console
```

Comment:
```suggestion
... code-block:: console

```

pretty sure you need the newline here.

Reply:
god damn I hate RST 🙈 😂

Reply:
It does take a bit of learning. I do like how it enforces a consistent convention around whitespace usage in the source. That kind of document formatting can be unwieldy in markdown.